core/input/input.cpp

```diff
@@ -1365,8 +1365,9 @@ void Input::parse_mapping(String p_mapping) {
 
 		JoyButton output_button = _get_output_button(output);
 		JoyAxis output_axis = _get_output_axis(output);
-		ERR_CONTINUE_MSG(output_button == JoyButton::INVALID && output_axis == JoyAxis::INVALID,
-				vformat("Unrecognized output string \"%s\" in mapping:\n%s", output, p_mapping));
+		if (output_button == JoyButton::INVALID && output_axis == JoyAxis::INVALID) {
+			print_verbose(vformat("Unrecognized output string \"%s\" in mapping:\n%s", output, p_mapping));
+		}
```

Comment:
I noticed thanks to a static analyzer that we should have kept a `continue;` here.
The further logic will be broken as it won't be able to assign a valid `outputType` to the mapping. (I'm making a PR.)